client/state/notices/middleware.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * External dependencies
+ */
+import { translate } from 'i18n-calypso';
+import { truncate } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { successNotice, errorNotice } from 'state/notices/actions';
+import { getSitePost } from 'state/posts/selectors';
+import {
+	POST_DELETE_FAILURE,
+	POST_DELETE_SUCCESS,
+	POST_SAVE_SUCCESS
+} from 'state/action-types';
+
+export const OBSERVERS = {
```

Comment:
> a class with static-only methods

Aha, @nylen - you have touched on another pet-peeve of mine :wink: Our PHP code is littered with static-only classes. That is, state-encapsulating containers which contain no state. I'm pretty sure that the only reason we have those in PHP all over the place is the general lack of namespacing for so long. At least here in JS we don't have that problem.

Point: If a class only has static methods, then we should probably eliminate the class and make those methods all functions instead.